docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -0,0 +1,108 @@
+as the authoritative copy of some of your indices. The high reliability of the
+snapshot repository removes the need to keep multiple copies of their data in
+your cluster purely for resiliency. {es} makes a copy of a searchable snapshot
+on the nodes in the cluster to reduce the performance impact and costs of
+accessing the snapshot repository.
+
+With {search-snaps-cap} you may be able to halve your cluster size without
+increasing the risk of data loss or reducing the amount of data exposed to
+searches. Put differently, {search-snaps-cap} may allow you to expose twice as
+much data to searches for a given cluster size.
+
+=== Using searchable snapshots
+
+A searchable snapshot can be searched just like any other index.
+{search-snaps-cap} are often used to access a large archive of historical data,
+for which searches may sometimes be complex and time-consuming.
+<<async-search>> is particularly useful for these long-running searches.
+
+The shards of searchable snapshots are also allocated just like shards of any
+other index. You can, for instance, use <<shard-allocation-filtering>> to
+restrict these shards to a subset of your nodes.
+
+Normally you will use {search-snaps-cap} via the
+<<ilm-searchable-snapshot,searchable snapshots ILM action>> which automatically
+and transparently converts your index into a searchable snapshot when it
+reaches the `cold` ILM phase. If you already have some snapshots that you want
+to search, you can also use the <<searchable-snapshots-api-mount-snapshot>> to
+manually mount them as searchable snapshots.
+
+You must not delete a snapshot while any of its indices are mounted as a
+searchable snapshot. However, most snapshots are taken as a backup and contain
+a large number of indices, most of which will not be mounted as searchable
+snapshots. If you mount just one of these indices then you may not delete that
+snapshot. Therefore we recommend that you use the <<clone-snapshot-api>> to
+cheaply create a clone of a snapshot where the clone contains just the index
+you want to mount. This will mean you can delete the multiple-index snapshot,
+reducing the size of your snapshot repository, without losing access to any
+mounted indices.
+
+We recommend that you <<indices-forcemerge,force-merge>> indices to a single
+segment per shard before mounting them as searchable snapshots. Each read from
+a snapshot repository takes time and costs money, and the fewer segments there
+are the fewer reads are needed to restore the snapshot.
+
+By default a searchable snapshot has `number_of_replicas` set to `0`. You can
+increase the number of replicas if desired, for instance if you want to perform
+more concurrent searches of these shards.
+
+If a node fails while holding some zero-replica searchable snapshot then there
+will be a brief window of time before {es} allocates these shards elsewhere.
+During this window of time the cluster health will be `red` and searches that
```

Comment:
I believe the cluster will currently only be yellow unless we worked on this recently?

Reply:
Ah yes now that the recovery source is always "snapshot" this should be the case. Hedging my bets in c43841dc1d59ac7dd814a3332818501619996617 by saying "not green".